src/scripts.js

```diff
@@ -22,7 +21,10 @@ const filterBar = document.querySelector('.filter-section-js');
 const mainView = document.querySelector('.main-view-container-js');
 const searchRecipesButton = document.getElementById('searchRecipes');
 const filterButton = document.getElementById('filterButton');
-// const antipasti = document.getElementById('antipasti')
+const homeButton = document.querySelector('.home-button');
+const searchBar = document.getElementById('searchInput');
```

Comment:
in reference to the not above, I think IDs are fine without the "-js" since we don't use ids in our css file. But it gets hard to keep tack if a class is for css, js, or both. So I think it seems helpful to have separate css-specific and js-specific classes, even if that means having two very similar class names for one element.